### PR TITLE
Add workflow execution and testing

### PR DESCRIPTION
**Pull request by MarkGibbons**
# Description

Describe what this change achieves

## Issues Resolved

Fixes issue #2

## Check List

- [ ] All tests pass. See TESTING.md for details.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+name: ci
+
+on:
```
Comment (ramereth) [inline .github/workflows/ci.yml:4]: This removes this warning.
```suggestion
"on":
```

File: kitchen.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 driver:
   name: vagrant
 driver_config:
-  require_chef_omnibus: 13.6.4
+  require_chef_omnibus: 15
```
Comment (ramereth) [inline kitchen.yml:5]: This should get converted to product_name and product_version

Comment (MarkGibbons) [reply kitchen.yml:5]: thank you.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (MarkGibbons) [reply .github/workflows/ci.yml:4]: Cool, I have that in a bunch of places and no idea what it meant.

**Pull request merged**